src/modules/verifiable-credentials/storage-client.ts

```diff
@@ -22,4 +32,80 @@ export default class VCStorageClient {
 
     return data;
   }
+
+  async store(
+    credentials: VerifiableCredential<CredentialSubject>[]
+  ): Promise<StoreVcResult[]> {
+    const extractId = (credential: VerifiableCredential<CredentialSubject>) => {
+      const isUuid = credential.id.search('urn:uuid:') > -1;
+      if (isUuid) {
+        return credential.id.split('urn:uuid:').pop() || v4();
+      }
+      return v4();
+    };
+
+    const messages = await Promise.all(
+      credentials.map(async (credential) => {
+        const data = Buffer.from(JSON.stringify(credential), 'utf8').toString(
+          'base64'
+        );
+
+        const message: WebNodeWriteMessage = {
+          data,
+          descriptor: {
+            method: 'CollectionsWrite',
+            objectId: extractId(credential),
+            schema: 'https://www.w3.org/2018/credentials#VerifiableCredential',
+            dateCreated: new Date(credential.issuanceDate).getTime(),
+            datePublished: Date.now(),
+            dataFormat: 'application/ld+json',
+            cid: await this.generateCid(credential),
+          },
+        };
+
+        return message;
+      })
+    );
+
+    const request: WebNodeRequestObject = {
+      requestId: v4(),
+      target: 'vp',
+      messages,
+    };
+
+    const { data: response } = await this._http.post<
+      WebNodeResponseObject<[WebNodeWriteMessage]>
+    >(`/`, request, {
+      baseURL: 'webnode',
+    });
+
+    if (response?.status?.code && response.status.code >= 400) {
+      throw new Error(response.status.message || 'Unknown error');
+    }
+
+    if (!response?.replies) {
+      throw new Error('No replies in response');
+    }
+
+    return response?.replies.map((replay) => {
+      if (replay?.status?.code && replay.status.code >= 400) {
```

Comment:
Minor typo, should be `reply`
```suggestion
    return response?.replies.map((reply) => {
      if (reply?.status?.code && reply.status.code >= 400) {
```